src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -114,10 +114,13 @@
                 foundSubscriptionList.add(subscription);
                 LOGGER.debug("Subscription [{}] fetched successfully.", subscriptionName);
             } catch (SubscriptionNotFoundException e) {
-                LOGGER.error("Subscription not found: {}",subscriptionName , e);
+                LOGGER.error("Subscription not found: {}", subscriptionName);
+                LOGGER.debug("Subscription not found traceback", e);
```

Comment:
Add colon after traceback. Or put the stacktrace on a new line

Reply:
fixed.